.docker/images/govcms8/settings/production.settings.php

```diff
@@ -12,14 +12,27 @@
 // Don't show any error messages on the site (will still be shown in watchdog)
 $config['system.logging']['error_level'] = 'hide';
 
-// Expiration of cached pages on Varnish to 15 min
+// Expiration of cached pages on Varnish to 15 min.
 $config['system.performance']['cache']['page']['max_age'] = 900;
 
-// Aggregate CSS files on
+// Aggregate CSS files on.
 $config['system.performance']['css']['preprocess'] = 1;
 
-// Aggregate JavaScript files on
+// Aggregate JavaScript files on.
 $config['system.performance']['js']['preprocess'] = 1;
 
-// Disabling stage file proxy on production, with that the module can be enabled even on production
+// Disabling stage file proxy on production, with that the module can be enabled
+// even on production.
 $config['stage_file_proxy.settings']['origin'] = false;
+
+// Configure Environment indicator.
+$config['environment_indicator.indicator']['bg_color'] = 'red';
+$config['environment_indicator.indicator']['name'] = 'Public Domain';
+
+if (!empty($_SERVER['HTTP_HOST'])) {
+  $http_host = $_SERVER['HTTP_HOST'];
+  if (preg_match('/(?<!www)\.govcms\.gov\.au/i', $http_host)) {
```

Comment:
Wondering if we can differentiate the actual govcms website from other customer websites?

For example:
`if (preg_match('/(?<!www)\.\.govcms\.gov\.au/i', $http_host)) {`

Would match anysite.govcms.gov.au and not govcms.gov.au, though given `www` is forced we might be able to dismiss the edge-case.